plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestingH2JdbcClient.java

```diff
@@ -109,6 +110,13 @@ public Optional<String> getTableComment(ResultSet resultSet)
         return Optional.empty();
     }
 
+    @Override
+    public void setColumnComment(ConnectorSession session, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        // do not throw when invoked, however do not allow to set non-empty comment until the connector supports setting column comments
+        verify(comment.isEmpty(), "This connector does not support setting column comments");
```

Comment:
This isn't correct anyway. 
`setColumnComment` invoked with empty comment should unset existing comment, so the `verify`'s safety is an illusion. 


what about:
```
// Ignore (not fail) for testing purposes.
```

without a `verify`?

Reply:
> This isn't correct anyway.

true.

> unset existing comment 

existing comment does not exist, because `the connector does not support setting column comments`.
And for the similar case with table comment `getTableComment` always returns `Optional.empty()`, so it's in tact. 

> without a verify

I'd left it as it is, because if I test `setColumnComment`, I'd not test it as `setColumnComment(Optional.empty())`, but with some value, and this could led to some higher expectations from tests which use this function.

Reply:
> > unset existing comment
> 
> existing comment does not exist, because `the connector does not support setting column comments`.

it could exist if H2 supports column comments.

Trino is not the only gate to the underlying db



> > without a verify
> 
> I'd left it as it is, because if I test `setColumnComment`, I'd not test it as `setColumnComment(Optional.empty())`, but with some value, and this could led to some higher expectations from tests which use this function.

Fine. Leave it but please make it clear it's not correct, but just as a reminder.

Reply:
I was thinking about it. I was thinking that setting empty comment is fine, for connectors that do not support it. However I was wrong:
 - It is a difference between empty comment and null comment
 - H2 may change and we could give a wrong impression that support it.

Reply:
Yes, it's controversial. So better trade-off is not to test caching for addComment.

Reply:
As agreed, left it as it is now.